src/server/game/Spells/SpellMgr.cpp

```diff
@@ -3265,3245 +3265,3952 @@ void SpellMgr::LoadSpellCustomAttr()
-            case 71838:
-            case 71839:
-                spellInfo->AttributesEx2 |= SPELL_ATTR2_CANT_CRIT;
-                spellInfo->AttributesEx4 |= SPELL_ATTR4_IGNORE_RESISTANCES;
-                break;
-            // Alchemist's Stone
-            case 17619:
-                spellInfo->AttributesEx3 |= SPELL_ATTR3_DEATH_PERSISTENT;
-                break;
-            // Gnomish Death Ray
-            case 13278:
-            case 13280:
-                spellInfo->EffectImplicitTargetA[0] = TARGET_UNIT_TARGET_ENEMY;
-                break;
-            // Stormchops
-            case 43730:
-                spellInfo->EffectImplicitTargetA[1] = 1;
-                spellInfo->EffectImplicitTargetB[1] = 0;
-                break;
-            // Savory Deviate Delight (transformations), allow to mount while transformed
-            case 8219:
-            case 8220:
-            case 8221:
-            case 8222:
-                spellInfo->Attributes &= ~SPELL_ATTR0_UNAFFECTED_BY_INVULNERABILITY;
-                break;
-            // Clamlette Magnifique
-            case 72623: // drink triggered spell
-                spellInfo->EffectBasePoints[EFFECT_0] = spellInfo->EffectBasePoints[EFFECT_1];
-                break;
-            // Compact Harvest Reaper
-            case 4078:
-                spellInfo->DurationIndex = 6; // 10 minutes
-                break;
-            // Dragon Kite, Tuskarr Kite - Kite String
-            case 45192:
-                spellInfo->rangeIndex = 6; // 100yd
-                break;
-            // Frigid Frostling, Infrigidate
-            case 74960:
-                spellInfo->EffectRadiusIndex[EFFECT_0] = 9; // 20yd
-                break;
+    // Stormchops
+    ApplySpellFix({ 43730 }, [](SpellEntry* spellInfo)
+    {
+        spellInfo->EffectImplicitTargetA[1] = 1;
+        spellInfo->EffectImplicitTargetB[1] = 0;
+    });
 
+    // Savory Deviate Delight (transformations), allow to mount while transformed
+    ApplySpellFix({ 8219, 8220, 8221, 8222 }, [](SpellEntry* spellInfo)
+    {
+        spellInfo->Attributes &= ~SPELL_ATTR0_UNAFFECTED_BY_INVULNERABILITY;
+    });
 
-            // ///////////////////////////////////////////
-            // ////////////////EVENTS/////////////////////
-            // ///////////////////////////////////////////
+    // Clamlette Magnifique
+    ApplySpellFix({ 72623 }, [](SpellEntry* spellInfo)
+    {
+        spellInfo->EffectBasePoints[EFFECT_0] = spellInfo->EffectBasePoints[EFFECT_1];
+    });
 
-            //////////////////////////////////////////
-            ////////// BREWFEST
-            //////////////////////////////////////////
-            // Apple Trap
-            case 43450:
-                spellInfo->EffectImplicitTargetA[0] = TARGET_UNIT_SRC_AREA_ENEMY;
-                spellInfo->EffectImplicitTargetB[0] = TARGET_UNIT_CASTER;
-                spellInfo->Effect[0] = SPELL_EFFECT_DUMMY;
-          
```

Comment:
missing brackets

Reply:
that is removed code O.o

Reply:
Ah I see now...
This was code I actually didn't touch but github diff is bad and does weird shit :D